lib/foreman/renderer/scope/macros/host_template.rb

```diff
@@ -30,14 +30,22 @@ def host_puppet_classes
             host.puppetclasses
           end
 
-          def host_param_true?(name)
+          def host_param_true?(name, default_value = false)
             check_host
-            host.params.has_key?(name) && Foreman::Cast.to_bool(host.params[name])
+            if host.params.has_key?(name)
+              Foreman::Cast.to_bool(host.params[name])
+            else
+              default_value
+            end
           end
 
-          def host_param_false?(name)
+          def host_param_false?(name, default_value = true)
             check_host
-            host.params.has_key?(name) && Foreman::Cast.to_bool(host.params[name]) == false
+            if host.params.has_key?(name)
+              Foreman::Cast.to_bool(host.params[name]) == false
+            else
+              default_value
```

Comment:
This will now return `true` if the param doesn't exist, where before it would return `false`.

Reply:
This still needs fixing - the default value should be false in this function as well to maintain compatibility